Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -1139,25 +1141,45 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
             m_expressionStack.constructAndAppend(Type { TypeKind::Ref, TypeInformation::get(typeDefinition) }, result);
             return { };
         }
-        case GCOpType::ArrayGet: {
+        case GCOpType::ArrayGet:
+        case GCOpType::ArrayGetS:
+        case GCOpType::ArrayGetU: {
```

Comment:
I think the cases for `ArrayGetS`/`ArrayGetU` are missing in the unreachable code parsing below (around line 2147).